docs/components/table.md

```diff
@@ -1,1124 +0,0 @@
-# Table
```

Comment:
issue: If we're just deprecating the component, we shouldn't delete the docs file.

You can put a "danger" note at the top explaining that it's deprecated and to use KTableData instead

Reply:
Restored the component docs page, only did not add it to the sidebar nav.

Reply:
It needs to be in the sidebar nav as well; just add the deprecation message to the top

Reply:
it's still used in our apps for a while so we don't want to hide documentation for existing instances. Deprecated doesn't mean no longer used